README.md

```diff
@@ -20,6 +20,7 @@ Command-line tools for Ringpop
     status       Status of members in ring
     partitions   Show partition information of a ring
     top          General membership information
+    reap         Remove the faulty nodes from this nodes membership
```

Comment:
good catch for heal not being here. I will add it again and figure out why it got removed again, cause I added it here in the first commit on the branch.